templates/tab/ts/default/.env.teamsfx.local

```diff
@@ -0,0 +1,7 @@
+# Following variables will be overwritten by TeamsFx
```

Comment:
@jidddddd Please help advice the wording part. Whether we should use `Following variables are generated by TeamsFx` to align with local-debug, or we should use something like `Following variables will be overwritten by TeamsFx` as the values are all empty after scaffold.

Reply:
@formulahendry can we change it to `TeamsFx will overwrite the following variable values when running debug`
@swatDong can we update the source code and doc for this comment as well?

Reply:
Updated in https://github.com/OfficeDev/TeamsFx/pull/6283